public/sass/messaging.scss

```diff
@@ -198,7 +198,8 @@ h2.optionsTitle {
 		font-family: $serif-stack;
 		font-size: 1rem;
 		padding-top: 1rem;
-		height: 5rem;
+		height: 5.5rem;
+		resize: none;
```

Comment:
Yeb! One of the new properties in the working-draft. Supported by Chrome, Safari and Firefox.